### ISSUE TITLE
Failed to convert a .pb file to a .lite file where there is a custom lite op sin

### ISSUE THREAD

**Issue by cambwang** [tensorflow/tensorflow#15414]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 14.04.5 LTS 
- **TensorFlow installed from (source or binary)**:binary
- **TensorFlow version (use command below)**:1.4.0
- **Python version**: 2.7.6
- **Bazel version (if compiling from source)**: 0.8.0
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem

I followed "[How to use custom operators](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/custom_operators.md)" to create a .pb file and "sin.cc" in //tensorflow/contrib/lite/kernel/. Then I added "TfLiteRegistration* Register_SIN();" and "AddCustom("Sin", Register_SIN());" in "register.cc"
But when I used the bazel command to covert the pb file, the sin op can not be converted

Here is the command I used:
bazel build //tensorflow/contrib/lite/toco:toco
bazel-bin/tensorflow/contrib/lite/toco/toco --input_file=tftest/sin.pb --input_format=TENSORFLOW_GRAPHDEF --output_file=tftest/sin.tflite --output_format=TFLITE --inference_type=FLOAT --input_array=input --input_shape=1 --output_array=output
Here is the corresponding ERROE information:
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If you have a custom implementation for them you can disable this error with --allow_custom_ops. Here is a list of operators for which you will need custom implementations: Sin.

I tried to use ----allow_custom_ops, but it did not work. Here is the ERROR information:
Converting unsupported operation: Sin

I think I have to modify more files, but I do not know which files I should modify and how to modify. Could you please give a detailed demo?
Thx

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by ailias** [comment]
Have you solved this problem yet ? I met same error now,  I modified model.h, schema.fbs, BUILD, register.cc and operator.cc similar with Cast op, but I got the error after running ios example:
Didn't find custom op for name 'xxx'
Registration failed.

**Comment by arogozhnikov** [comment]
I think any working example for conversion behind 'using a pretrained mobilenet' would help (e.g. wasn't able to convert shufflenet from onnx).

**Comment by trangtv57** [comment]
any one have solution now ?
